### ISSUE TITLE
Feature: Adding date separation sections in directory flattening and searches

### ISSUE THREAD

**Issue by ltctceplrm** [bpatrik/pigallery2#689]
**Is your feature request related to a problem? Please describe.**
My directories are organized according to /YYYY/MM but when I use directory flattening on an entire year or if I search for a person/place I can't quickly see when a photo was taken without clicking on it and then pressing the "i" button.

**Describe the solution you'd like**
I'd like to have an option to visually separate the photos into sections per date (and maybe an option per month as well).

Basically like google photo has:
![](https://i.imgur.com/GpxR597.png)

In fact that sidebar for easy timeline navigation would be a great feature too but I suppose I should open a separate feature request for that?

**Comment by johnnyadora** [comment]
This feature was added in #706 so I'm going to close my issue, the new feature works great by the way.

**Comment by ltctceplrm** [comment]
This feature was added in #706 so I'm going to close my issue, the new feature works great by the way.
